### ISSUE TITLE
For Generator, is dropout used in training and testing  the same?

### ISSUE THREAD

**Issue by lilu1996** [KupynOrest/DeblurGAN#135]
I found that in the definition of G  you used the description of "**not** opt.no_dropout", that means "true", but in the testing process you didn't use model.eval(). I'm confused.

**Comment by lilu1996** [comment]
the argument '--no_dropout' is not included, so "not opt.no_dropout" represents "true", right?

**Comment by lilu1996** [comment]
#

**Comment by lilu1996** [comment]
#34 I got it
